test/css/test_parser.rb

```diff
@@ -63,8 +63,8 @@ def test_to_a
       end
 
       def test_has
-        assert_xpath  "//a[b]", @parser.parse("a:has(b)")
-        assert_xpath  "//a[b/c]", @parser.parse("a:has(b > c)")
+        assert_xpath  "//a[.//b]", @parser.parse("a:has(b)")
+        assert_xpath  "//a[.//b/c]", @parser.parse("a:has(b > c)")
```

Comment:
But it should match just `a`, that's the point of the `:has` css pseudo class. Otherwise you could just use a normal descendant selection, not `has`. Or am I missing something?